frontend/src/metabase/admin/settings/components/SettingsUpdatesForm.jsx

```diff
@@ -54,7 +55,7 @@ export default class SettingsUpdatesForm extends Component {
               target="_blank"
```

Comment:
ExternalLink will set target based on whether the host matches. You could drop this without changing behavior.

Reply:
Good tip, thanks @paulrosenzweig. I've adjusted it accordingly.